foreign_cc/repositories.bzl

```diff
@@ -12,6 +12,7 @@ def rules_foreign_cc_dependencies(
         cmake_version = "3.23.2",
         make_version = "4.4",
         ninja_version = "1.11.1",
+        meson_version = "0.63.0",
```

Comment:
There are now newer meson releases but lets land this first and then we can work on bumping the release to latest.